### ISSUE TITLE
Moving away from adapt_framework

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#3325]
### Subject of the issue
I'd like to propose that with v6 of adapt_framework, we switch the main build tools to a new repo called `adapt-framework` or some similar idea. The outcome of which would be to remove the large history of commits and files from the download process and greatly speed up the install.

Please let me know your thoughts.

**Comment by marqpdx** [comment]
Aside from losing old commit history, would anything else be lost?

Plus, if there will be a tagged repo for v5, nothing is being permanently lost, correct?

**Comment by oliverfoster** [comment]
Correct, adapt_framework will still be a repo, it just won't be the main working repo for the framework build tools as it is at the moment.
